packages/expo/src/provider/singleton/singleton.ts

```diff
@@ -0,0 +1,9 @@
+import { Clerk } from '@clerk/clerk-js/headless';
+
+import { createClerkInstance } from './createClerkInstance';
+
+/**
+ * @deprecated Use `getClerkInstance` instead. `Clerk` will be removed in the next major version.
+ */
+export { clerk } from './createClerkInstance';
```

Comment:
I'd drop this, if we are cutting a new major

Reply:
As we discussed we will not drop that in this version as we don't want to introduce any API changes, the bump of the major is only for React Native and Expo version bumps, to be more clear which version the SDK supports